examples/provider/provider.tf

```diff
@@ -233,5 +314,22 @@ resource "dcloud_ip_nat_rule" "ip_nat_rule"{
 resource "dcloud_vm_nat_rule" "vm_nat_rule"{
   topology_uid = dcloud_topology.test_topology.id
   nic_uid = dcloud_vm.vm1.network_interfaces[1].uid
-  east_west = false
+  east_west = true
+}
+
+resource "dcloud_inbound_proxy_rule" "inbound_proxy_rule"{
+  topology_uid = dcloud_topology.test_topology.id
+  nic_uid = dcloud_vm.vm2.network_interfaces[1].uid
+  target_vm_name = dcloud_vm.vm1.name
```

Comment:
I think target_vm_name should be an "output only" value - it's fully dependent on the selected NIC

Reply:
Yes, I thought the same initially but to my surprise it was required, else the request is throwing a "400 Bad Request".